<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>****************************************************************************
**
** Copyright (C) 2021 Donna Whisnant, a.k.a. Dewtronics.
** Contact: http://www.dewtronics.com/
**
** This file is part of the frsky_sport_tool Application.
**
** GNU General Public License Usage
** This file may be used under the terms of the GNU General Public License
** version 3.0 as published by the Free Software Foundation and appearing
** in the file gpl-3.0.txt included in the packaging of this file. Please
** review the following information to ensure the GNU General Public License
** version 3.0 requirements will be met:
** http://www.gnu.org/copyleft/gpl.html.
**
** Other Usage
** Alternatively, this file may be used in accordance with the terms and
** conditions contained in a signed written agreement between you and
** Dewtronics.
**
****************************************************************************</comment>
 <class>CConfigDlg</class>
 <widget class="QDialog" name="CConfigDlg">
  <property name="windowTitle">
   <string>Communications Configuration</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="framePortMapping">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout3">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboFirmwarePort"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboDataConfigPort"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblFirmwarePort">
        <property name="text">
         <string>Firmware Port:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboFirmwarePort</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblDataConfigPort">
        <property name="text">
         <string>Data/Config Port:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboDataConfigPort</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkBoxFirmwareLogTxEchos">
        <property name="text">
         <string>Log Tx Echos</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="chkBoxDataConfigLogTxEchos">
        <property name="text">
         <string>Log Tx Echos</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="grpBoxSport2">
     <property name="title">
      <string>S.port #2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblSport2SerialPort">
        <property name="text">
         <string>&amp;Serial Port:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1SerialPort</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport2DataBits"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport2SerialPort"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblSport2StopBits">
        <property name="text">
         <string>S&amp;top Bits:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1StopBits</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSport2BaudRate">
        <property name="text">
         <string>&amp;Baud Rate:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1BaudRate</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport2StopBits"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport2Parity"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblSport2DataBits">
        <property name="text">
         <string>&amp;Data Bits:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1DataBits</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <spacer name="verticalSpacer2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblSport2Parity">
        <property name="text">
         <string>&amp;Parity:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1Parity</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport2BaudRate"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpBoxSport1">
     <property name="title">
      <string>S.port #1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout1">
      <item row="0" column="0">
       <widget class="QLabel" name="lblSport1SerialPort">
        <property name="text">
         <string>&amp;Serial Port:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1SerialPort</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport1DataBits"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport1SerialPort"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblSport1StopBits">
        <property name="text">
         <string>S&amp;top Bits:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1StopBits</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSport1BaudRate">
        <property name="text">
         <string>&amp;Baud Rate:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1BaudRate</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport1StopBits"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport1Parity"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblSport1DataBits">
        <property name="text">
         <string>&amp;Data Bits:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1DataBits</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <spacer name="verticalSpacer1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblSport1Parity">
        <property name="text">
         <string>&amp;Parity:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboSport1Parity</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboSport1BaudRate"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboSport1SerialPort</tabstop>
  <tabstop>comboSport1BaudRate</tabstop>
  <tabstop>comboSport1Parity</tabstop>
  <tabstop>comboSport1DataBits</tabstop>
  <tabstop>comboSport1StopBits</tabstop>
  <tabstop>comboSport2SerialPort</tabstop>
  <tabstop>comboSport2BaudRate</tabstop>
  <tabstop>comboSport2Parity</tabstop>
  <tabstop>comboSport2DataBits</tabstop>
  <tabstop>comboSport2StopBits</tabstop>
  <tabstop>comboFirmwarePort</tabstop>
  <tabstop>chkBoxFirmwareLogTxEchos</tabstop>
  <tabstop>comboDataConfigPort</tabstop>
  <tabstop>chkBoxDataConfigLogTxEchos</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CConfigDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CConfigDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
